android/app/build.gradle

```diff
@@ -65,6 +63,30 @@ android {
         disable 'InvalidPackage'
     }
     namespace 'dk.analog.digitalclipcard'
+
+    flavorDimensions "app"
+
+    productFlavors {
+        development {
+            dimension "app"
+            applicationId "dk.analogio.analog.dev"
```

Comment:
What's the reason for the different application-id's instead of just adding suffixes for the flavour?

Reply:
Xcode claimed the app id DK.AnalogIO.DigitalCoffeeCard.Dev (which is the app id for iOS with the .Dev suffix) while I was trying to build the app locally. I don't know how to unclaim the app id so I just went with an entirely different app id for development (production app ids are untouched)

Reply:
Makes sense, all good